src/main/java/com/google/idea/perf/TracerController.kt

```diff
@@ -297,6 +294,19 @@ class TracerController(
         return ProgressManager.getInstance().runProcess(computable, progress)
     }
 
+    private fun reloadAutocompleteClasses() {
```

Comment:
Right now this is only called in the constructor of TracerController.
Do you plan to call it elsewhere as well?

Reply:
Yup, I'm planning to call this periodically later on